randovania/data/json_data/prime1/Chozo Ruins.txt

```diff
@@ -540,28 +540,46 @@ Asset id: 1226570426
       Any of the following:
           Grapple Beam and Heat-Resisting Suit
           All of the following:
-              Space Jump Boots and Heated Rooms Damage ≥ 70
+              Space Jump Boots
+              Any of the following:
+                  Heat-Resisting Suit
+                  Heat Runs (Beginner) and Heated Rooms Damage ≥ 70
               Any of the following:
-                  Movement (Intermediate) and Lava Damage ≥ 220
+                  All of the following:
+                      Movement (Intermediate)
+                      Any of the following:
+                          # Standable terrain method: https://www.youtube.com/watch?v=eklXYB2qJ9E
+                          Standable Terrain (Intermediate) or Lava Damage ≥ 220
                   Scan Visor and Combat/Scan Dash (Advanced)
   > Pickup (Power Bomb Expansion)
-      Power Bomb and Morph Ball and Heated Rooms Damage ≥ 60
+      All of the following:
+          Power Bomb and Morph Ball
+          Any of the following:
+              Heat-Resisting Suit
+              Heat Runs (Beginner) and Heated Rooms Damage ≥ 60
 
 > Door to Meditation Fountain; Heals? False; Spawn Point
   * Normal Door to Meditation Fountain/Door to Magma Pool
   > Door to Training Chamber Access
       Any of the following:
           Grapple Beam and Heat-Resisting Suit
           All of the following:
-              Space Jump Boots and Heated Rooms Damage ≥ 99
+              Space Jump Boots
               Any of the following:
-                  Gravity Suit and L-Jump (Intermediate) and Movement (Intermediate) and Standable Terrain (Intermediate) and Lava Damage ≥ 50
+                  All of the following:
+                      # https://www.youtube.com/watch?v=eklXYB2qJ9E
```

Comment:
this video shows the wrong direction

Reply:
Also, I would argue that the gravity suit L-Jump is easier in trick requirements, and so is worth modeling the lava damage for.

(L-Jump intermediate, Standable beginner)

Reply:
I've addressed both of these comments in the latest commit